### PR TITLE
Integrating third party platform

### PR DESCRIPTION
**Pull request by PippaClarkGDS**
## Why

Need documentation for users integrating using third-party platforms. 

## What

New documentation gives guidance on third-party platforms and client_secret_post token authentication method.

## Technical writer support

This has been through tech writer review. 

## How to review

- check out the `integrating-third-party-platform` branch
- build the content using the new `./build-with-docker.sh` script and ensure that the html-proofer step at the end of the build runs cleanly and does not generate any errors that halt the build
- preview the content using `./preview-with-docker.sh` to ensure that the new content works in the context of the overall site

## Changelog

Changelog has been updated. 

## Confirm

- [y] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [y] Where there is any overlap I have updated or opened a PR for corresponding changes

### CODE REVIEW

**Comment by pauldougan** [comment]
@PippaClarkGDS I have maxe two commits to help resolve our pipeline problem as follows:

- added the two urls that were causing issues with html-proofer to the list of exceptions in `config.rb` so that they don't cause the pipeline to break if the proofer complains, I will raise a jira ticket to capture more investigation into this
- a new convenience script `build-with-docker.sh` modelled after `preview-with-docker.sh` to run the middleman build and html-proofer from a Docker container to assist with diagnostics, it will also be a useful script when checking prs local ahead of the final merge

In my view it is better to retain the proofer in the build stage with two exceptions, than to entirely remove the proofer or to prevent it finding legitimate problems that will halt the pipeline.

File: source/before-integrating/integrating-third-party-platform.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+---
+
+
+# Integrating third-party platforms with GOV.UK One Login
+
+If you use a third-party platform (Software as a Service such as Salesforce or Microsoft Power Platform, or an identity provider such as Amazon Cognito or ForgeRock) to integrate with GOV.UK One Login, you might experience issues or specific limitations during integration.
+
+Contact GOV.UK One Login at govuk-one-login@digital.cabinet-office.gov.uk ([govuk-one-login@digital.cabinet-office.gov.uk](mailto:govuk-one-login@digital.cabinet-office.gov.uk) will open a link to your mail client) if you’re using a third-party platform to integrate with GOV.UK One Login. 
+
+GOV.UK One Login will update this page regularly with detailed information on integrating with third-party platforms
```
Comment (sabrinaharris) [inline source/before-integrating/integrating-third-party-platform.html.md.erb:15]:
```suggestion
GOV.UK One Login will update this page regularly with information on integrating with third-party platforms.
```
Is this sentence helpful to users? We're not saying what 'regularly' is and the page itself already sets the expectation we'll provide information about integrating with third-party platforms!

Comment (PippaClarkGDS) [reply source/before-integrating/integrating-third-party-platform.html.md.erb:15]: good point ... will amend @sabrinaharris

**Review by sabrinaharris — APPROVED** [review]
🚀

**Pull request merged**